jvb/src/test/kotlin/org/jitsi/videobridge/ConferenceTest.kt

```diff
@@ -94,7 +75,7 @@ private val newConfigOctoEnabled = """
         octo {
             enabled = true
             bind-address = 127.0.0.1
-            bind-port = 4096
+            bind-port = 54096
```

Comment:
Should we use a random port here?

Reply:
I don't know of an easy way to get a random free port without binding to it. Are you worried about this failing if 54096 is  not free?

Reply:
That and running tests in parallel.  I wasn't thinking we needed to get too fancy...just literally using a random (valid) number.

Reply:
The more tests we run in parallel the higher the probability we hit a used port.

Reply:
Yeah, I think the ideal thing would be to add a hook in OctoRelayService for a "UdpTransportFactory": a function it uses to create a UDP transport so we could have it create a fake which doesn't bind at all.